app/views/patients/_pledge.html.erb

```diff
@@ -35,8 +35,23 @@
       <% if current_tenant.pledge_generation_config.present? %>
         <p class="row"><%= t('patient.pledge.step_three.generated', fund: current_tenant.name) %></p>
         <p class="row">
-          <%= t('patient.pledge.step_three.downloads') %> <%= fax_service %>
+          <%= t('patient.pledge.step_three.downloads') %>
         </p>
+
+        <% if patient.clinic&.email_for_pledges.present? %>
+          <p class="row">
+            <%= clinic_pledge_email(patient) %>
+          </p>
+        <% else %>
+          <% if patient.clinic&.fax.present? %>
+            <p class="row">
+              Clinic fax:&nbsp;<strong><%= clinic_pledge_fax(patient) %></strong>
+            </p>
+          <% end %>
+          <p class="row">
+            Fax service:&nbsp;<%= fax_service %>
```

Comment:
these `&nbsp;` make me nervous. better way to handle? the issue is that leading/trailing spaces aren't observed when concatenating tags – in the case of the fax #, I'm `<strong>`ing the number itself, so the space gets collapsed; in the case of `fax_service`, we have a link, so the space between the words and the `<a>` is collapsed.

Reply:
@colinxfleming this look ok?